docs/iceberg-rest-service.md

```diff
@@ -147,6 +147,51 @@ If you have a JDBC Iceberg catalog prior, you must set `catalog-backend-name` to
 You must download the corresponding JDBC driver to the `iceberg-rest-server/libs` directory.
 :::
 
+#### Multi catalog support
+
+The Gravitino Iceberg REST server supports multiple catalogs and offers a configuration-based catalog management system.
+
+| Configuration item                           | Description                                                                                                      | Default value               | Required | Since Version |
+|----------------------------------------------|------------------------------------------------------------------------------------------------------------------|-----------------------------|----------|---------------|
+| `gravitino.iceberg-rest.catalog-provider`    | The implementation of IcebergTableOpsProvider defines how the Iceberg REST catalog server gets Iceberg catalogs. | `config-based-provider`     | No       | 0.7.0         |
```

Comment:
Catalog provider class name, you can develop a class that implements `xxx` and add the corresponding jar file to the Iceberg REST service classpath directory.